crates/snapbox/src/path.rs

```diff
@@ -257,15 +260,14 @@ impl PathDiff {
                         crate::Data::read_from(&actual_path, None).map_err(Self::Failure)?;
 
                     let expected = crate::Data::read_from(&expected_path, None)
-                        .map(|d| d.map_text(crate::utils::normalize_lines))
+                        .map(|d| d.normalize(NormalizeNewlines))
                         .map_err(Self::Failure)?;
 
-                    if let (Some(expected), format) = (expected.as_str(), expected.format()) {
```

Comment:
side note: The dropping of the `if`s would have ideally been a separate commit so they don't get lost as we need to consider how safe this is.

I *think* it should be safe

Reply:
I moved the if statements into their own commit, so we can look at them individually